tooling/bundle-content-scripts.ts

```diff
@@ -45,6 +45,7 @@ buildContentScript(
     getFilesInDir(`${sourceDir}/js/common/api/shared`, /\.js$/, false),
     getFilesInDir(`${sourceDir}/js/common/api/key-server`, /\.js$/, false),
     getFilesInDir(`${sourceDir}/js/common/api/account-servers`, /\.js$/, false),
+    getFilesInDir(`${sourceDir}/js/common/api/authentication`, /\.js$/, false, true),
```

Comment:
it'll be better to create separate folder `/api/authentication/google` and store `google-oauth.ts` there (like we already do in `email-provider` folder), instead of adding `reverseFilesInDir` parameter.
as it's possible that we'll have another `..-oauth.ts` file which alphabetically will be after `oauth.ts` and `reverseFilesInDir` won't work in this case.

Reply:
Ok thanks. I think I get you the idea. Will do - thank you.